python/dask_cudf/dask_cudf/tests/test_distributed.py

```diff
@@ -1,10 +1,12 @@
+# Copyright (c) 2020-2022, NVIDIA CORPORATION.
+
 import numba.cuda
 import pytest
 
 import dask
 from dask import dataframe as dd
 from dask.distributed import Client
-from distributed.utils_test import loop  # noqa: F401
+from distributed.utils_test import cleanup, loop  # noqa: F401
```

Comment:
Is `cleanup` used below somewhere?

Reply:
We had to import due to a CI failure, I think the reason is similar to: https://github.com/rapidsai/dask-cuda/pull/924/files

Reply:
Just tested removing `cleanup` locally and got errors around `Cluster failing to start`, so will need it.

Reply:
Thanks for checking 🙏 Strange. Maybe we should raise a Distributed issue about this?